test/uniport-tests.js

```diff
@@ -0,0 +1,502 @@
+/* -------------------------------- REQUIRES -------------------------------- */
+
+var async       = require("async");
+var assert      = require('assert');
+var assert      = require('assert');
+var UInt160     = require("ripple-lib").UInt160;
+var Remote      = require('ripple-lib').Remote;
+var Server      = require('ripple-lib').Server;
+var Request     = require('ripple-lib').Request;
+var testutils   = require("./testutils");
+var config      = testutils.init_config();
+var http        = require('http');
+var request     = require('request');
+
+/* --------------------------------- CONFIG --------------------------------- */
+
+// So we can connect to websocket ports
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+var configurations = {
+  // All the tests run on one invokation of rippled. It's possible for you to:
+  use_own_rippled : false,
+
+  // We have one main websocket connection, that we inherit from
+  // `testutils.build_setup`. We use that, configured on a non admin, non ssl
+  // port, to determine readiness for tests on the other ports. This Remote's
+  // trace is configured in the normal way, however we can set trace for each of
+  // the test connections on the various ports.
+  remote_trace : false,
+
+  // There's a bug where we can't subscribe to the admin ports requiring
+  // credentials, even though subscribe has Role::USER. `Remote` always issues a
+  // subscribe command after connect(). We can just add the credentials to let
```

Comment:
I think instead of "We can just add", you might want to say, "Setting this flag adds the credentials to let..."